src/tests/services.test.js

```diff
@@ -0,0 +1,40 @@
+import {
+  updateCountryNames,
+
+} from '../services';
+
+describe('covid19 tracker services', () => {
```

Comment:
- [ ] Perfectly, you wrote a test for your project! I would suggest you write a little more about it. The main one is, your tests are failing you can see the screenshot:

![Screen Shot 2022-04-22 at 10 52 29 PM](https://user-images.githubusercontent.com/69011963/164770745-bc33bdf4-5596-4adb-aae0-6b18cee93dd7.png)

Reply:
Thanks for this suggestion.I have added more tests and fixed the error